SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using SportsHub.AppService.Authentication.Models.DTOs;
+using SportsHub.AppService.Services;
+using System.Text.Json;
+
+namespace SportsHub.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ArticleController : ControllerBase
+    {
+        private readonly IArticleService service;
+        private readonly IMapper mapper;
+
+        public ArticleController(IArticleService service, IMapper mapper)
+        {
+            this.service = service;
+            this.mapper = mapper;
+        }
+
+        [HttpGet("GetArticleByTitle")]
+        public async Task<IActionResult> GetArticleByTitleAsync(string title)
```

Comment:
I am not sure that we need GetArtickleByTitle here, it looks like too much.

Reply:
@MBocharskyi What do you think about not returning an ActionResult but some response model? Or we will improve it with the middleware of exception?

Reply:
It is better to use `ActionResult<ResponseModel>`

Reply:
Like this `public async Task<ActionResult<ArticleResponseDTO>> GetArticleByTitleAsync(string title)`?